packages/clerk-js/src/ui/utils/captcha.ts

```diff
@@ -0,0 +1,29 @@
+import { loadScript } from '../../utils';
+
+interface RenderOptions {
+  sitekey: string;
+  callback?: (token: string) => void;
+}
+
+declare global {
+  export interface Window {
+    turnstile: { execute: (container?: string | HTMLElement | null, params?: RenderOptions) => void };
+  }
+}
+
+export const SCRIPT_URL = 'https://challenges.cloudflare.com/turnstile/v0/api.js?render=explicit';
```

Comment:
❓ Do we need to export SCRIPT_URL ?

Reply:
@panteliselef you are right we don’t need to